packages/taucmdr/cli/commands/trial/tests/test_create.py

```diff
@@ -91,7 +91,10 @@ def test_heap_usage_memory_alloc_sample(self):
         stdout, stderr = self.assertCommandReturnValue(0, select_cmd, ['sample'])
         self.assertIn("Selected experiment 'targ1-app1-sample'", stdout)
         self.assertFalse(stderr)
-        self.assertManagedBuild(0, CC, ['-g', '-no-pie'], 'matmult.c')
+        ccflags = ['-g']
+        if (not util.which('nvc')) :
```

Comment:
Add a comment that explains why this check is necessary, such as:

```suggestion
        # The NVHPC compilers don't accept the -no-pie option
        if (not util.which('nvc')) :
```

Reply:
If they have a different position independent executable option we should enable that---some OSes (I think Ubuntu) default to PIEs and this breaks instrumentation.

Reply:
If NVHPC has such an option, it is not documented.